### ISSUE TITLE
how to implement bee-queue in a nodejs cli app?

### ISSUE THREAD

**Issue by nllorca** [bee-queue/bee-queue#516]
I am trying to do something and I am not finding the way. I have a cli app in nodeJS. This application must run, and in its execution my intention is that it can launch some processes (in a queue), but not wait for the response of said processes.

I have tried it with bee-queue but when I do the main app (cli app), it is frozen, and it never finishes its execution.

Is there a way to get it with bee-queue?

In summary what I need is:

- Run a nodeJS cli app
- The cli app runs and queues a number of processes
- The cli app terminates (and the processes continue to do their work)

Thanks!

**Comment by compwright** [comment]
In your main app you will queue a job, and in a separate process run the code to consume and process the job.
